doc/source/users/macro_logging.rst

```diff
@@ -0,0 +1,21 @@
+.. highlight:: python
+   :linenothreshold: 5
+   
+.. currentmodule:: sardana.macroserver.macros.logging
+
+
+.. _sardana-macros-logging:
+
+=============
+Macro Logging
+=============
+
+The MacroServer allows to write into a log file messages from the class responsible for
```

Comment:
This part about "the class responsible running for running the macros" is too specific for the user. Especially as the first sentence describing this feature. I would eventually mention something about additional log entries that enclose each macro execution e.g. when it starts and when it ends. What do you think?